### ISSUE TITLE
Scan Directory/File not recognizing .3ds or .cxi ROM Files

### ISSUE THREAD

**Issue by LOuroboros** [libretro/RetroArch#6387]
## Description
If I use the Scan Directory or Scan File options and use them to scan a folder that only contains 2 .3ds and 1 .cxi ROM Files, those ROMs do not show up anywhere inside RetroArch.

### Expected behavior
Well, I thought they would appear somewhere, as some sort of shortcut.
I don't know, something [like this](https://youtu.be/DGlRGXq_9Ic?t=388).

### Steps to reproduce the bug
1. Scan Directory/File
2. Select a folder with nothing but .3ds and/or .cxi ROM Files
3. Scan this directory.
4. Go back to RetroArch's Main Menu and see by yourself that your games are nowhere to be found.

### Version/Commit
- RetroArch: Git d170cb5469

### Environment information
- OS: Windows 8.1 _(64 Bits)_

**Comment by RobLoach** [comment]
Could you post the CRCs of those files? Which games are they? Would like to compare against [libretro-database](https://github.com/libretro/libretro-database).

**Comment by LOuroboros** [comment]
@RobLoach I don't know if I'm doing this correctly because I never had to check the CRC of a file.
I got them with a free software called "RapidCRC".
Then I got the same results after trying to get them with a different software called "GUI File Signature Widget".

Dragon Ball Z: Extreme Butouden
CRC: 1DFB08DC

Pokémon Omega Ruby
CRC: 732F7AD3

The Legend of Zelda: Ocarina of Time 3D.
CRC: 6C9329D0

I just compared them, and while OoT3D seems to be missing, the CRC of my ROMs don't match the ones shown in https://github.com/libretro/libretro-database/blob/e83f0bef1951e4cd9e8417ea02ba0c7879e2aa52/metadat/no-intro/Nintendo%20-%20Nintendo%203DS%20(DLC).dat

I suppose that is the issue and RetroArch only recognizes 1 specific kind of dump, is that correct?
